tools/wpt/wpt.py

```diff
@@ -44,7 +44,11 @@ def load_commands():
         abs_path = os.path.join(wpt_root, path, "commands.json")
         base_dir = os.path.dirname(abs_path)
         with open(abs_path, "r") as f:
-            data = json.load(f)
+            try:
+                data = json.load(f)
+            except Exception:
+                sys.stderr.write(f"ERROR: Failed reading commands file {abs_path}\n")
```

Comment:
Can this happen now but not before? Or did you add this to catch your JSON syntax errors while preparing this? If so, maybe it can be dropped now :)